news/2023/2023-12-07-winter-fanart-contest-results.md

```diff
@@ -0,0 +1,105 @@
+
+[![](https://assets.ppy.sh/contests/189/banners/ase4an.jpg)](https://assets.ppy.sh/contests/189/winners/ase4an.png)
+
+**980** voters had hearts in their eyes for [**ase4an**](https://osu.ppy.sh/users/12646324)'s rendition of a festive pippi, Chirou and Mocha set before a dazzling array of fireworks. Someone's got a present she's eager to pass along, by the looks of things.
+
+### peppy's Picks
+
+It is said that in the hallowed halls of the osu! office (creative deficiencies in its name notwithstanding) that if one presses their ear to the wall adjacent to the Dean "peppy" Herbert's room during working hours around the advent of a fanart contest, they will hear one of three things: an Otographic Arts mix from several months ago, furious keyboard chattering, or irate mumbling about not having enough votes.
+
+To address one of these issues, the Picks were born. And so it is that every fanart contest since, the man himself declares a selection of his personal favourites shall make it to the main menu, for they are beloved in his eyes, and so too shall they be beloved in ours. Also, they're just generally awesome and we get more art to look at. Everybody wins!
+
+[![](https://assets.ppy.sh/contests/189/banners/AyoLie.jpg)](https://assets.ppy.sh/contests/189/winners/AyoLie.png)
+
+[![](https://assets.ppy.sh/contests/189/banners/Christi_Anna.jpg)](https://assets.ppy.sh/contests/189/winners/Christi_Anna.png)
+
+[![](https://assets.ppy.sh/contests/189/banners/Maqmoon.jpg)](https://assets.ppy.sh/contests/189/winners/Maqmoon.png)
+
+[![](https://assets.ppy.sh/contests/189/banners/sleepysheeps.jpg)](https://assets.ppy.sh/contests/189/winners/sleepysheeps.png)
+
+[![](https://assets.ppy.sh/contests/189/banners/HellothereEm.jpg)](https://assets.ppy.sh/contests/189/winners/HellothereEm.png)
+
+---
+
+A big thank you to everyone who took the time out of their day to enter or vote (or perhaps both)! It's the buy-in from the community that helps make these contests so great, and we hope you had fun participating in whatever way you could.
+
+As always, the top 3 finalists have received a special bespoke profile badge to acknowledge their victory, and every finalist (including the Picks!) has taken a toasty sum of 3 months of osu!supporter home with them over the holidays.
+
+[You can download a zip file containing all the finalist Winter 2023 Fanart Contest entries here!](https://assets.ppy.sh/contests/189/WinterFanart2023Finalists.zip) If you like something you see and it's not on this post, why not try reaching out to the artist or checking their socials? You can check out the full skew of the voting results over at the [contest listing](https://osu.ppy.sh/community/contests/189), if you like.
```

Comment:
"skew"? do you mean "skewer" or something?

```suggestion
[You can download a zip file containing all the finalist Winter 2023 Fanart Contest entries here!](https://assets.ppy.sh/contests/189/WinterFanart2023Finalists.zip) If you like something you see and it's not on this post, why not try reaching out to the artist or checking their socials? You can check out the full skewer of voting results over at the [contest listing](https://osu.ppy.sh/community/contests/189), if you like.
```

otherwise 

```suggestion
[You can download a zip file containing all the finalist Winter 2023 Fanart Contest entries here!](https://assets.ppy.sh/contests/189/WinterFanart2023Finalists.zip) If you like something you see and it's not on this post, why not try reaching out to the artist or checking their socials? You can check out the full ensemble of voting results over at the [contest listing](https://osu.ppy.sh/community/contests/189), if you like.
```

Reply:
it's a statistics term but skewer is unintentionally immensely funnier so let's use that